content/docs/components/hyperparameter.md

```diff
@@ -369,4 +295,4 @@ recall=0.55
 precision=0.5
 ```
 
-Katib collects all logs of metrics.
+Katib will then periodically launch CronJobs to collect metrics from pods.
```

Comment:
Suggestion: remove the future tense "will then"; just say "Katib periodically launches CronJobs..." Otherwise, I am a little confused and I wonder: Does Katib collect these metrics (via CronJobs) by default, or do I have to do something specific to enable this metrics collection?